test/functional/data/invalid_txs.py

```diff
@@ -130,6 +132,21 @@ def get_tx(self):
         assert MIN_STANDARD_TX_NONWITNESS_SIZE - 1 == 64
         return tx
 
+# reject a transaction that contains a witness
+# but doesnt spend a segwit output
```

Comment:
doesnt -> doesn’t [missing apostrophe in contraction “doesn't”]

Reply:
Taken in a626caec847c1c542c663be6664229857d2bf220